x-pack/plugins/apm/public/components/app/transaction_details/distribution/index.tsx

```diff
@@ -118,6 +118,14 @@ export function TransactionDistribution({
           detailTab={detailTab as TransactionTab | undefined}
           waterfallFetchResult={waterfallFetchResult}
           traceSamplesFetchResult={traceSamplesFetchResult}
+          showCriticalPath={showCriticalPath}
+          onShowCriticalPathChange={(nextShowCriticalPath) => {
+            push(history, {
+              query: {
+                showCriticalPath: nextShowCriticalPath ? 'true' : 'false',
```

Comment:
Can you use `toString` instead?
```suggestion
                showCriticalPath: nextShowCriticalPath.toString(),
```

Reply:
I see your point, but I think I have a slight preference for being explicit here.